sciencebeam_utils/tools/check_file_list.py

```diff
@@ -48,23 +48,32 @@ def map_file_list_to_file_exists(file_list):
         return list(executor.map(FileSystems.exists, file_list))
 
 
-def format_file_exists_results(file_exists):
+def format_file_list(file_list):
+    return '%s' % file_list
+
+
+def format_file_exists_results(file_exists, file_list):
     if not file_exists:
         return 'empty file list'
     file_exists_count = sum(file_exists)
     file_missing_count = len(file_exists) - file_exists_count
+    files_missing = [s for s, exists in zip(file_list, file_exists) if not exists]
     return (
-        'files exist: %d (%.0f%%), files missing: %d (%.0f%%)' %
```

Comment:
if this removes decimals explictly, good for pinning down the format

Reply:
yes it does.